dummy_robot/dummy_robot_bringup/launch/dummy_robot_bringup_launch.xml

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <node pkg="dummy_map_server" exec="dummy_map_server" output="screen" />
+  <node pkg="robot_state_publisher" exec="robot_state_publisher" output="screen" >
+    <param name='robot_description' value="$(command 'cat $(find-pkg-share dummy_robot_bringup)/launch/single_rrbot.urdf')"/>
```

Comment:
@p-vega why single quotes for the `name` attribute value?

Reply:
@hidmic you're right, they should be double quotes. I'm sorry for the typo, I'll change it and I'll add a new commit.